swap_meet/vendor.py

```diff
@@ -1,2 +1,60 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+        except ValueError:
+            print("That item is not in the inventory.")
+            return False
+        else:
+            return item
+    
+    def get_by_category(self, category):
+        items_with_category = [item for item in self.inventory if item.category == category]
+        return items_with_category
```

Comment:
Great to see you're practicing list comprehensions! Very succinct way to handle building up the list 🙂

Since we don't perform any other operations with `items_with_category`, we could also directly return the result of the comprehension:
```suggestion
        return [item for item in self.inventory if item.category == category]
```